### ISSUE TITLE
Wrong formatting with JSON inside strings

### ISSUE THREAD

**Issue by gimmi** [dzhibas/SublimePrettyJson#166]
This is my config `Pretty JSON.sublime-settings`

```json
{
    "indent": 3
}
```

When I try to format this:

```json
{
  "createOptions": "{\"HostConfig\":{\"PortBindings\": {\"443/tcp\":[{ \"HostPort\": \"443\" }]}}}"
}
```

I get this

![image](https://user-images.githubusercontent.com/6589/151161330-511cb45b-6819-49f2-9f7a-debb0dc7fe81.png)

**Comment by DrNickBailey** [comment]
I get the same issue. I have json which when you prettify it it adds these rogue line breaks which then prevents minification.

Surely there could be an easy test case to add around this.

**Comment by TheSecEng** [comment]
I've seen that this is caused by `brace_newline` setting. In the meantime you can set this to `False`

**Comment by TheSecEng** [comment]
Fixed in 1.3.0
